js_modules/dagster-ui/packages/ui-core/src/instance/InstanceConcurrencyKeyInfo.tsx

```diff
@@ -0,0 +1,641 @@
+            >
+              <Heading>
+                <Box flex={{direction: 'row', gap: 8, alignItems: 'center'}}>
+                  <div>
+                    <Link to="/deployment/concurrency">
+                      {flagPoolUI ? 'Pools' : 'Concurrency keys'}
+                    </Link>
+                  </div>
+                  <div>/</div>
+                  <div>{concurrencyKey}</div>
+                </Box>
+              </Heading>
+              <Box flex={{direction: 'row', gap: 8, alignItems: 'center'}}>
+                <QueryRefreshCountdown refreshState={refreshState} />
+                <Popover
+                  position="bottom-right"
+                  content={
+                    <Menu>
+                      <MenuItem
+                        icon="delete"
+                        text="Delete"
+                        intent="danger"
+                        onClick={() => setShowDelete(true)}
+                      />
+                    </Menu>
+                  }
+                >
+                  <Button icon={<Icon name="expand_more" />} />
+                </Popover>
+              </Box>
+            </Box>
+            <Box padding={{vertical: 16, horizontal: 24}}>
+              <Subheading>{flagPoolUI ? 'Pool info' : 'Concurrency info'}</Subheading>
+            </Box>
+            <Box padding={{bottom: 24}}>
+              <MetadataTableWIP style={{marginLeft: -1}}>
+                <tbody>
+                  {flagPoolUI ? (
+                    <tr>
+                      <td style={{verticalAlign: 'middle'}}>Granularity</td>
+                      <td>Op</td>
+                    </tr>
+                  ) : null}
+                  <tr>
+                    <td style={{verticalAlign: 'middle'}}>Limit</td>
+                    <td>
+                      <Box flex={{justifyContent: 'space-between', alignItems: 'center'}}>
+                        <div>
+                          {concurrencyLimit.slotCount ? (
+                            concurrencyLimit.slotCount
+                          ) : concurrencyLimit.limit === null ? (
+                            <>&mdash;</>
+                          ) : (
+                            concurrencyLimit.limit
+                          )}
+                        </div>
+                        <Button icon={<Icon name="edit" />} onClick={() => setShowEdit(true)}>
+                          Edit limit
+                        </Button>
+                      </Box>
+                    </td>
+                  </tr>
+                </tbody>
+              </MetadataTableWIP>
+            </Box>
+            <Box
+              padding={{vertical: 16, horizontal: 24}}
+              flex={{direction: 'row', alignItems: 'center', justifyContent: 'space-between'}}
+            >
+              <Subheading>In progress</Subheading>
+            </Box>
+            <Box style={{marginLeft: -1}}>
```

Comment:
What does it look like without these?

Reply:
There's a left-border for all the table rows, so it looks like an extra thick border.  I couldn't figure out how to disable that, so I just cheated 😭

Reply:
<img width="777" alt="Screenshot 2025-01-23 at 12 11 01 PM" src="https://github.com/user-attachments/assets/adb4ed5d-b738-45cc-959c-b1a8a9b972f9" />

Reply:
Yeah, I guess we have that elsewhere too. Oh well. Probably worth it for me to go in and fix these.

Could have sworn we had resolved that in some way at some point, since all of our tables tend to be full-width...